8.2-arm/config/supervisord/supervisord.conf

```diff
@@ -0,0 +1,34 @@
+[supervisord]
+nodaemon=true
+
+[program:logstash]
+command = /logstash-entrypoint.sh logstash -f /etc/logstash/conf.d/
+autostart=false
```

Comment:
Why disable autostart logstash? So that the docker cannot find logstash in command.  Could you tell me how to start it manually? Thanks.

Reply:
@timbotetsu can you please address @bndynet's issue?